CHANGES.md

```diff
@@ -24,6 +24,8 @@ To be released.
    - Added `BlockChain<T>.FindNextHashes(BlockLocator, BlockHash?, int)` method.
    - Added `BlockChain<T>.Fork(BlockHash, bool)` method.
    - Added `BlockChain<T>.GetBlockLocator(int)` method.
+ -  Added `IStore.GetCanonicalGenesisBlock(HashAlgorithmGetter)` method.
+    [[#2162], [#2171]]
```

Comment:
Since adding a new method to an existing interface enforces classes implementing it to implement the new method, this change should be list in the backward-incompatible API changes.